pkgs/applications/editors/bluej-bin/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, makeWrapper, unzip, jetbrains
+}:
+let
+  version = "414";
```

Comment:
version can be passed on to mkDerivation

Reply:
I'm not sure how. When I put `version` into mkDerivation like this
```
stdenv.mkDerivation {
  version = "414";
  name = "bluej-bin";
  src = fetchurl {
    url = "https://www.bluej.org/download/files/BlueJ-linux-${version}.deb";
    sha512 = "c9c5c905d6dcf4388fe1423179be836ad8d35a4d197e53706a5764d9192c439aa60b80a28b87222b9f16ed1534d7a7768d924fc06ddd95107aea3cb00570c22a";
  };
```
I get
`error: undefined variable 'version' `

Reply:
Clarification:
```
stdenv.mkDerivation rec {
  version = "414";
  pname = "bluej-bin";
  src = fetchurl {
    url = "https://www.bluej.org/download/files/BlueJ-linux-${version}.deb";
    sha512 = "c9c5c905d6dcf4388fe1423179be836ad8d35a4d197e53706a5764d9192c439aa60b80a28b87222b9f16ed1534d7a7768d924fc06ddd95107aea3cb00570c22a";
  };
```

Added missing `rec` and changed `name` to `pname`.